[2022-02-03 20:34:55,570] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_2.download_dataset_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-03 20:34:55,606] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_2.download_dataset_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-03 20:34:55,608] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:34:55,609] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 20:34:55,610] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:34:55,650] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-01-02 06:00:00+00:00
[2022-02-03 20:34:55,661] {standard_task_runner.py:52} INFO - Started process 4944 to run task
[2022-02-03 20:34:55,670] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_2', 'download_dataset_task', 'scheduled__2019-01-02T06:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2.py', '--cfg-path', '/tmp/tmpaoi_rdk1', '--error-file', '/tmp/tmpy6bw0_1r']
[2022-02-03 20:34:55,673] {standard_task_runner.py:77} INFO - Job 27: Subtask download_dataset_task
[2022-02-03 20:34:55,803] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_2.download_dataset_task scheduled__2019-01-02T06:00:00+00:00 [running]> on host 985445ce9491
[2022-02-03 20:34:55,974] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_2
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-02-03 20:34:55,980] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 20:34:55,982] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd']
[2022-02-03 20:34:55,998] {subprocess.py:85} INFO - Output:
[2022-02-03 20:34:56,043] {subprocess.py:89} INFO - /tmp/***tmpg2fkj12k
[2022-02-03 20:34:56,046] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 20:34:56,119] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_2, task_id=download_dataset_task, execution_date=20190102T060000, start_date=20220203T203455, end_date=20220203T203456
[2022-02-03 20:34:56,207] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 20:34:56,291] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
